### ISSUE TITLE
new release with most recent changes?

### ISSUE THREAD

**Issue by pmihaylov-nydig** [dnaeon/gru#54]
Last release is from 2017, but there have been more recent changes since, which address various bugs & new functionality. Perhaps makes sense to make a new release to avoid go getting master directly?

**Comment by dnaeon** [comment]
Hey @pmihaylov-nydig ,

I no longer work actively on this project, and that's the reason it hasn't seen a new release.

**Comment by pmihaylov-nydig** [comment]
yeah I was only wondering if a release can be bundled with whatever's latest in master, rather than look for any new changes to be submtited?
